### ISSUE TITLE
Add Wiki on Github page and link existing Docs.

### ISSUE THREAD

**Issue by Stephanevg** [Stephanevg/PSHTML#17]
Add documentation to the project by implementing the following:

1) Create Github Wiki page. --> https://github.com/Stephanevg/PSHTML/wiki
2) Link the existing Docs which are located here --> https://github.com/Stephanevg/PSHTML/tree/master/docs

**Comment by sebastiaanspeck** [comment]
I would like to assign this issue to me

**Comment by sebastiaanspeck** [comment]
It is a bit of a pain to create Github Wiki page and then create a pull request for it. Could you create a simple Wiki-page first? Then I can clone that .wiki locally and change it

**Comment by Stephanevg** [comment]
Awesome, it is all your @sebastiaanspeck !

I have created the first Page as per your request.

Concerning the context:

What is important for us, is to have the docs that @ChendrayanV  added in his PR ( [here](https://github.com/Stephanevg/PSHTML/tree/master/docs)) in that page.

I think the best view would be something like this:

Having a Side bar, where every individual .md file in the \docs folder would be available. When clicked, it should display that page.

The landing page should be something that explains a bit the intent of the Wiki Section, and show cases some very easy and quick samples on how to create a web page using PSHTML (simple copy/ paste from the main readme.md will suffice in the first place I think).

I am unfamiliar on how this should / could be constructed, but it would be great if we can have it smartly structured in a folder structure in the repo. Either directly under Docs/ or perhaps under a new folder 'Wiki' perhaps?

All of the above, are just 'suggestions' and if yo see a better way of implementing this, please go ahead. (Again, the imporant thing for us, is to give an easy access to the help of every cmdlet documentation).

Let me know if you need more infos!

**Comment by sebastiaanspeck** [comment]
I need permissions to change the Wiki.

If I fork the repo, I lose the Wiki, so I need to commit to the repo-wiki directly.

![screenshot 2018-10-02 at 13 32 27](https://user-images.githubusercontent.com/12570668/46346320-eca4c880-c647-11e8-92cd-eb2d5c900963.png)

**Comment by Stephanevg** [comment]
Hi, hmmm I am not sure how to fix this @sebastiaanspeck 
Have you tried this perhaps? https://help.github.com/articles/adding-and-editing-wiki-pages-locally/

**Comment by sebastiaanspeck** [comment]
I will try that tomorrow, but if I clone it, the home.md you created, it disappears if I fork it and want to clone the wiki. As said, I will try again tomorrow

**Comment by Stephanevg** [comment]
Hi again @sebastiaanspeck ,
I actually managed to clone it using the following link:
https://github.com/Stephanevg/PSHTML.wiki.git

I modified it quickly locall, and reuploaded it . 
- git add *
- git commit m "My commit message"
- git push

Have you already tried that, or do you have another problem?

**Comment by Stephanevg** [comment]
Worse case, you can create a new one. There is nothing in anyways.

**Comment by sebastiaanspeck** [comment]
I know. But right now I got the Wiki on my fork (not looking perfect, but it's a start), but I can't create a pull request to merge my Wiki into the PSHTML wiki... As said before, I need to get write-access to commit the Wiki. Or you have to take a look at the fork of mine and copy that to your local machine and commit it to the wiki.git

**Comment by Stephanevg** [comment]
Hi, @sebastiaanspeck  I have changed a setting, which should allow you to create a PR on the Wiki repo now.
Could you please try again?

**Comment by sebastiaanspeck** [comment]
Yes. I can't find the way on how to do a PR, but I can go to the Wiki and add/edit pages. This should be enough
